src/netgroup.h

```diff
@@ -70,8 +86,19 @@ class NetGroupManager {
      * re-bucketed.
      *
      * This is initialized in the constructor, const, and therefore is
-     * thread-safe. */
-    const std::vector<std::byte> m_asmap;
+     * thread-safe. m_asmap can either point to m_loaded_asmap which holds
+     * data loaded from an external file at runtime or it can point to embedded
+     * asmap data.
+     */
+    const std::span<const std::byte> m_asmap;
+    std::vector<std::byte> m_loaded_asmap;
```

Comment:
In commit dc0c4310bae257236d3cd6a06de8071276d4548b

You mention that this dual ownership structure is done to avoid holding the asmap in memory twice. Where is this currently the case? Does that only happen in test code at the moment?

Reply:
It was the case an earlier version of the code. I guess referencing something that isn't actually there anymore isn't something we should be doing in docs, even if it's implicitly. But in this case I think there should be some documentation in this regard so it clears up why we need `m_asmap` and `m_loaded_asmap` here.